guides/source/active_record_encryption.md

```diff
@@ -163,152 +238,157 @@ class Article < ApplicationRecord
 end
 ```
 
-### Ignoring Case
+### Ensuring Uniqueness with Encrypted Data
+
+Checking for uniqueness is only supported with deterministically encrypted data.
 
-You might need to ignore casing when querying deterministically encrypted data. Two approaches make accomplishing this easier:
+#### Unique Validations
 
-You can use the `:downcase` option when declaring the encrypted attribute to downcase the content before encryption occurs.
+In order to support unique validations, you'll need to enable extended queries.
+The default value for this configuration is false:
```

Comment:
I think it would be clearer if we can move the reasoning of using `extend_queries` on line 265 here, so there is less jumping of context?
Maybe something like:
```suggestion
In order to support unique validations, you'll need to enable extended queries. This makes sure we query for the downcased encrypted attribute (if used) and previous encryption schemes as well. The default value for this configuration is false:
```